lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -496,3 +531,85 @@ other helper function `boto3_tag_list_to_ansible_dict` to get an appropriate tag
 calling this function. Since the AWS APIs are not uniform (e.g. EC2 versus Lambda) this will work
 without modification for some (Lambda) and others may need modification before using these values
 (such as EC2, with requires the tags to unset to be in the form `[{'Key': key1}, {'Key': key2}]`).
+
+## Integration Tests for AWS Modules
+
+All new AWS modules should include integration tests to ensure that any changes in AWS APIs that
+affect the module are detected. At a minimum this should cover the key API calls and check the
+documented return values are present in the module result.
+
+For general information on running the integration tests see the [Integration Tests page of the
+Module Development Guide](http://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html).
+Particularly the [cloud test configuration section](http://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html#other-configuration-for-cloud-tests)
+
+The integration tests for your module should be added in `test/integration/targets/MODULE_NAME`.
+
+You must have the following in `test/integration/targets/MODULE_NAME/aliases`
+
+```
+cloud/aws
+posix/ci/cloud/group2/aws
```

Comment:
Contributors should be instructed to use the same group as existing AWS tests, which happens to be group4 currently. If we avoid referencing the group directly then it won't be out of date when the grouping changes.

Reply:
Thinking about how to make this clear. I think what I'll do is word it to be existing tests and link to one of the existing tests rather than including an inline example.

That way it's obvious what to copy without creating the documentation update overhead. Does that work?

Reply:
That works for me.